twa

```diff
@@ -78,34 +78,54 @@ function get_field {
       tolower(\$1) ~ /^${field}/ { print substr(\$0, index(\$0, \$2)) }" | tr -d '\r\n'
 }
 
+function output {
+  if [[ -n "${csv}" ]]; then
+    echo "${1},${2},${@:3}";
```

Comment:
Per `shellcheck`, this should probably be `${*:3}` here and below.

The explanation can also contain commas, so we probably need to double-quote it:

```bash
echo "${1},${2},\"${*:3}\""
```